webpack.config.js

```diff
@@ -1,22 +1,76 @@
+const fs = require('fs')
 const path = require('path')
-const srcPath = path.join(__dirname, 'src') + path.sep
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const marked = require('meta-marked')
 const widgetWebpack = require('materia-widget-development-kit/webpack-widget')
-const entries = widgetWebpack.getDefaultEntries()
+const rules = widgetWebpack.getDefaultRules()
+const copy = widgetWebpack.getDefaultCopyList()
+
+const srcPath = path.join(__dirname, 'src') + path.sep
+const outputPath = path.join(process.cwd(), 'build')
 
-entries['player.js'] = [
-	srcPath+'player.coffee',
-	srcPath+'draw.coffee'
-]
+const customCopy = copy.concat([
+	{
+		from: path.join(__dirname, 'src', '_guides', 'assets'),
+		to: path.join(outputPath, 'guides', 'assets'),
+		toType: 'dir'
+	},
+])
 
-entries['creator.js'] = [
-	srcPath+'creator.coffee',
-	srcPath+'draw.coffee',
-	srcPath+'spectrum.custom.js'
-]
+const entries = widgetWebpack.getDefaultEntries()
```

Comment:
`yarn build` is erroring because `entries` is declared twice. This line isn't required, as we're defining our `entires` object below and passing it into `widgetWebpack.getLegacyWidgetBuildConfig()` as part of the `options` object.